components/table/table-wrapper.js

```diff
@@ -381,6 +391,25 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		});
 	}
 
+	_checkSiblingSortableCells(c) {
+		const nodes = Array.from(c.childNodes);
+		const isSortableCell = nodes.some((element) => element.localName === 'd2l-table-col-sort-button');
+
+		if (isSortableCell) {
+			let siblings = 0;
+			nodes.forEach((element) => {
+				if (element.localName) siblings++;
+			});
```

Comment:
Is there a way to just get the count of number of items in the cell? Then we'd know if it was more than 1 then there are siblings.